test/units/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# pip packages required to use unit tests
```

Comment:
We already have a units requirements list:

https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt